ignite/metrics/kl_divergence.py

```diff
@@ -0,0 +1,102 @@
+from typing import Sequence
+
+import torch
+import torch.nn.functional as F
+
+from ignite.exceptions import NotComputableError
+from ignite.metrics.metric import Metric, reinit__is_reduced, sync_all_reduce
+
+__all__ = ["KLDivergence"]
+
+
+class KLDivergence(Metric):
+    r"""Calculates the mean of `Kullback-Leibler (KL) divergence
+    <https://en.wikipedia.org/wiki/Kullback%E2%80%93Leibler_divergence>`_.
+
+    .. math:: D_\text{KL}(\mathbf{p}_i \| \mathbf{q}_i) = \sum_{c=1}^C p_{i,c} \log \frac{p_{i,c}}{q_{i,c}}
+
+    where :math:`\mathbf{p}_i` and :math:`\mathbf{q}_i` are the prediction and ground true probability tensors.
```

Comment:
```suggestion
    where :math:`\mathbf{p}_i` and :math:`\mathbf{q}_i` are the prediction and ground truth probability tensors.
```

Reply:
According to the wiki page: https://en.wikipedia.org/wiki/Kullback%E2%80%93Leibler_divergence and also pytorch note on kl_div: https://pytorch.org/docs/stable/generated/torch.nn.KLDivLoss.html#torch.nn.KLDivLoss, in $D_{KL}(P \| Q)$, $P$ represents the data and $Q$ represents the model. Here in the docstring you have the opposite P for predictions and Q for targets. Should we use wiki's definitions?

Reply:
@kzkadc can you check above comment ?

Reply:
Indeed, I think so too.

Reply:
@vfdev-5 I swapped the order of the arguments to align with the KLDivLoss of pytorch. Could you check the change?